common/src/main/java/us/myles/ViaVersion/protocols/protocol1_13to1_12_2/packets/WorldPackets.java

```diff
@@ -242,6 +244,24 @@ else if (particleId == 36)
                             return;
                         }
 
+                        //Handle reddust particle color
+                        ifStatement:
```

Comment:
Does the label need to be here?

Otherwise great stuff :)

Reply:
It's for the break statement right here:
https://github.com/MylesIsCool/ViaVersion/pull/902/commits/bb4b5d3080865a4cc31013f90ca2a20041e08300#diff-1013ef390334502300ca050e5e285756R252